### ISSUE TITLE
not working anymore, after the release of quickshare

### ISSUE THREAD

**Issue by sisti00** [grishka/NearDrop#149]
after the release of the new quickshare app (= merge of Nearby Share with quickshare of samsung), NearDrop doesn't work anymore

**Comment by victor-falcon** [comment]
It's working for me from Phone to Computer, but not in the other way around

**Comment by Bennik2000** [comment]
It is working for me, too.

However, it fails for some arbitrary file types like *.csv files.

**Comment by dev-lcc** [comment]
same, it no longer works on QuickShare

**Comment by edgegithuber7** [comment]
Yes I got the same Issue Mac Can't discover Android device with Quick Share

**Comment by grishka** [comment]
If only someone actually offered any ideas how to fix it.

**Comment by dohun0310** [comment]
The only way to fix it is probably to recreate things like links in macOS.

https://github.com/Martichou/rquickshare

**Comment by SuperSharpShooter81** [comment]
I can send from Android to MacBook just fine using a Galaxy S23 and S23 Ultra.

When sending from MacBook to Android, neither of my Samsung devices appear. However, if I start Quick Share on my S23 (as if I'm attempting to send a photo), my S23 Ultra will appear available on the MacBook, and vice versa when I start Quick Share from my S23 Ultra, then the S23 will appear available on the Mac. 🤯

@grishka, not sure if this helps at all.
